JetNews/app/src/main/java/com/example/jetnews/ui/home/PostCardTop.kt

```diff
@@ -48,7 +48,7 @@ import com.example.jetnews.ui.themeTypography
 fun PostCardTop(post: Post) {
     // TUTORIAL CONTENT STARTS HERE
     Padding(16.dp) {
-        Column(crossAxisSize = LayoutSize.Expand) {
+        FlexColumn(crossAxisSize = LayoutSize.Expand) {
```

Comment:
Should still be a `Column`, `FlexColumn` and `Column` have been merged into one, using modifiers to define flexible behavior. (default is inflexible). Ditto elsewhere / for any `FlexRows`.